src/TestExample/Worm/Worm.svelte

```diff
@@ -1,30 +1,57 @@
 <script>
-  import worm from './worm.svg';
+	import worm from './worm.svg';
```

Comment:
Did the indent config change? I'm assuming this is using whatever is in .prettierrc, but maybe just check?

Reply:
Hmmm. This project has useTabs: true, but Aunty doesn't.

I reckon this file probably hasn't been touched since it was generated. Thus this change is converting spaces to tabs.

<img width="485" alt="image" src="https://github.com/abcnews/svelte-scrollyteller/assets/49600/c9d9d4ab-f19f-4a4e-98d5-34e51ba8962c">